src/SFA.DAS.EAS.Web.IntegrationTests/TestUtils/DataHelper/DbBuilder.cs

```diff
@@ -13,21 +15,38 @@ class DbBuilder
     {
         private readonly IHashingService _hashingService;
         private readonly IPublicHashingService _publicHashingService;
+        private EmployerAccountsDbContext _dbContext;
 
         public DbBuilder(
             DbBuilderDependentRepositories dependentRepositories,
             IHashingService hashingService,
-            IPublicHashingService publicHashingService)
+            IPublicHashingService publicHashingService,
+            EmployerAccountsDbContext dbContext)
         {
             DependentRepositories = dependentRepositories;
             _hashingService = hashingService;
             _publicHashingService = publicHashingService;
+            _dbContext = dbContext;
         }
 
         public DbBuilderContext Context { get;  } = new DbBuilderContext();
 
         public DbBuilderDependentRepositories DependentRepositories { get; }
 
+        public DbBuilder BeginTransaction()
+        {
+            _dbContext.Database.BeginTransaction();
+            return this;
+        }
+
+        public DbBuilder CommitTransaction()
+        {
+            if (_dbContext.Database.CurrentTransaction == null) return this;
+
+            _dbContext.Database.CurrentTransaction.Commit();
```

Comment:
I wondered whether it would be worthwhile guarding against errant calls? e.g. multiple calls to BeginTransaction and CommitingTransaction? Pretty sure errant calls will be faulted anyway by MSSQL but instinctively feel the class should be defending/handling this itself. However, this is a test helper class and simplicity > sophistication so perhaps as it is might be better.